backend/src/openarchiefbeheer/destruction/tests/test_serializers.py

```diff
@@ -445,6 +445,7 @@ def test_assign_author_as_reviewer(self):
 
 
 class DestructionListReviewSerializerTests(TestCase):
+    @override_settings(LANGUAGE_CODE="en")
```

Comment:
This shouldn't be needed if you mark the string as translatable in the assertion

Reply:
Does the test suit run with a gueranteed language of EN? If so: probably fine, but if it's dependent on settings. It might cause issues when content is actually translated right?

Reply:
It should be that if you mark the string as translatable in the assertion, then it should be in the same language as the string generated in the code (since they will share the setting).

Reply:
Ah, I didn't notice the tests were using a _() call as well... ☕